06.bowling_object/game.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+
+class Game
+  TOTAL_FRAMES = 10
+
+  def initialize(scores)
+    @scores = scores
+    @frames = generate_frames
```

Comment:
```suggestion
    @frames = generate_frames(scores)
```

インスタンス変数は寿命が長い変数なので極力減らした方が安全なプログラムになります。
https://bootcamp.fjord.jp/pages/311